Sources/Tracking/CustomerIO.swift

```diff
@@ -353,4 +367,30 @@ public class CustomerIO: CustomerIOInstance {
     ) {
         implementation?.screen(name: name, data: data)
     }
+
+    /**
+     Register a new device token with Customer.io, associated with the current active customer. If there
+     is no active customer, this will fail to register the device
+     */
+    public func registerDeviceToken(_ deviceToken: String) {
+        implementation?.registerDeviceToken(deviceToken)
+    }
+
+    /**
+     Delete the currently registered device token
+     */
+    public func deleteDeviceToken() {
+        implementation?.deleteDeviceToken()
+    }
+
+    /**
+     Track a push metric
+     */
+    public func trackMetric(
```

Comment:
These are public functions already. Just that the file which contained them changed. Levi might have more to add to this.

Reply:
These newly added functions are explained better [in this ticket](https://github.com/customerio/issues/issues/6934). We were already planning on doing this work of moving messaging push logic into Tracking module and I decided to include it in this PR because we would need to have both of these functions exist inside of a notification service extension: 

```swift
CustomerIO.initialize(...)
MessagingPush.initialize()
```

I was afraid that customers would forget to call `MessagingPush.initialize()` in a notification service extension and cause push metrics from rich push to not work. 

Since I knew we already made the decision to move the push metrics logic to the tracking module already, I decided the best time to do that was now in this PR. Close that ticket and fix this problem I am talking about here.